index.js

```diff
@@ -52,81 +20,28 @@ export const formatNumber: FormatNumberFn = (start, options) => {
   return `${options.prefix}${x1}${x2}${options.suffix}`;
 };
 
-export const startAnimation = (component: Component<*, *, *>) => {
+const startAnimation = (component) => {
   if (!(component && component.spanElement)) {
-    throw new Error(
-      'You need to pass the CountUp component as an argument!\neg. this.myCountUp.startAnimation(this.myCountUp);',
-    );
+    throw new Error('You need to pass the CountUp component as an argument!\neg. this.myCountUp.startAnimation(this.myCountUp);');
```

Comment:
It's tempting to clean stuff like this up, but I think you should adhere to the parent project as closely as possible.  Because we're going to either 1) be rebasing/merging this fork to keep up with updates from the parent project or 2) unforking this after we can use react 15.  Both situations are a lot simpler if the diff against the upstream is both minimal and meaningful.

That consideration applies to several spots in this PR, won't point them all out.

Reply:
Yeah I agree with Mr. Potato.  If something doesn't hurt to leave it in, we should just leave it in to make life easier down the road.